testdata/go/v3/memcached-operator/Makefile

```diff
@@ -136,34 +136,32 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
+LOCAL_BIN = $(shell pwd)/bin
+$(LOCAL_BIN):
+	mkdir -p $(LOCAL_BIN)
+
+CONTROLLER_GEN = $(LOCAL_BIN)/bin/controller-gen
+CONTROLLER_GEN_VERSION = v0.7.0
 .PHONY: controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0)
+controller-gen: $(CONTROLLER_GEN)
+$(CONTROLLER_GEN): $(LOCAL_BIN) ## Download controller-gen locally if necessary.
+	GOBIN=$(LOCAL_BIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_GEN_VERSION)
 
-KUSTOMIZE = $(shell pwd)/bin/kustomize
+KUSTOMIZE = $(LOCAL_BIN)/kustomize
```

Comment:
Why the Makefile here is diff of kb: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/Makefile? We ought to only add to add on top what is required for the specific SDK helpers. 

/hold